editor.planx.uk/src/pages/FlowEditor/components/Sidebar/Publish/PublishDialog.tsx

```diff
@@ -1,389 +1,342 @@
+          </Typography>
+          <InputLabel label="Summarise the changes" htmlFor="summary">
+            <ErrorWrapper
+              error={
+                showError ? "Provide a summary before publishing" : undefined
+              }
+              id="input-error-publishing-summary"
+            >
+              <Input
+                id="summary"
+                data-testid="publish-summary-input"
+                bordered
+                required
+                type="text"
+                name="summary"
+                value={summary}
+                multiline
+                rows={2}
+                onChange={(e) => changeInput(e)}
+              />
+            </ErrorWrapper>
+          </InputLabel>
+        </DialogContent>
+        <DialogFooterActions>
+          <Button onClick={handleBack}>Back</Button>
+          <Button
+            data-testid="publish-button"
+            color="primary"
+            variant="contained"
+            onClick={validateAndPublish}
+          >
+            Publish
+          </Button>
+        </DialogFooterActions>
+      </>
+    );
+  };
 
   return (
-    <Box sx={{ display: "flex", alignItems: "center" }}>
-      {atLeastOneFail ? (
-        <Cancel
-          color="error"
-          fontSize="large"
-          sx={{ minWidth: (theme) => theme.spacing(5.5) }}
-        />
-      ) : (
-        <CheckCircle
-          color="success"
-          fontSize="large"
-          sx={{ minWidth: (theme) => theme.spacing(5.5) }}
-        />
-      )}
-      <Typography
-        variant="body1"
-        component="div"
-        sx={{ display: "flex", flexDirection: "column" }}
-        gutterBottom
+    <Dialog
+      open={dialogOpen}
+      fullWidth
+      maxWidth="md"
+      onClose={() => {
+        setDialogOpen(false);
+        setActiveStep(0);
+      }}
+      aria-labelledby="alert-dialog-title"
+      aria-describedby="alert-dialog-description"
+      PaperProps={{
+        sx: {
+          height: "100%",
+          overflowY: "hidden",
+        },
+      }}
+    >
+      <Stepper
+        activeStep={activeStep}
+        sx={{
+          padding: 2,
+          px: 3,
+          backgroundColor: (theme) => theme.palette.background.default,
+          borderBottom: `1px solid`,
+          borderColor: (theme) => theme.palette.border.main,
+        }}
       >
-        {atLeastOneFail ? `Fix errors before publishing` : `Ready to publish`}
-        <Typography variant="caption">{formattedSummary}</Typography>
-      </Typography>
-    </Box>
+        {steps.map((label) => (
+          <Step key={label}>
+            <StepLabel
+              sx={(theme) => ({
+                fontWeight: "bold",
+                "& .MuiStepIcon-root.Mui-completed": {
+                  color: theme.palette.success.main,
+                },
+                "& .MuiStepIcon-root.Mui-active": {
+                  color: theme.palette.info.main,
+                },
```

Comment:
nit: If we import `stepIconClasses` we can refer to these directly instead of relying on strings which is better for maintainability. Feeling quite aware we're on MUI v5 and v7 is just released! 👀

```suggestion
                [`& .${stepIconClasses.completed}`]: { 
                  color: theme.palette.success.main 
                },
                [`& .${stepIconClasses.active}`]: { 
                  color: theme.palette.info.main
                },
```